src/tests/ftest/util/server_utils_params.py

```diff
@@ -319,15 +320,31 @@ def __init__(self, index=None):
             self.pinned_numa_node = BasicParameter(None)
             self.log_mask = BasicParameter(None, "DEBUG,RPC=ERR")
             self.log_file = LogParameter(log_dir, None, "daos_server.log")
-            self.env_vars = BasicParameter(
-                None,
-                ["ABT_ENV_MAX_NUM_XSTREAMS=100",
-                 "ABT_MAX_NUM_XSTREAMS=100",
-                 "DAOS_MD_CAP=1024",
-                 "FI_SOCKETS_MAX_CONN_RETRY=1",
-                 "FI_SOCKETS_CONN_TIMEOUT=2000",
-                 "DD_MASK=mgmt,io,md,epc,rebuild"]
-            )
+
+            # Set extra envariables for sockets provider
+            if default_provider is "ofi+sockets":
+                self.env_vars = BasicParameter(
+                    None,
+                    ["ABT_ENV_MAX_NUM_XSTREAMS=100",
+                     "ABT_MAX_NUM_XSTREAMS=100",
+                     "DAOS_MD_CAP=1024",
+                     "FI_SOCKETS_MAX_CONN_RETRY=5",
+                     "FI_SOCKETS_CONN_TIMEOUT=2000",
+                     "CRT_SWIM_RPC_TIMEOUT=10",
+                     "SWIM_PING_TIMEOUT=10000",
+                     "SWIM_PROTOCOL_PERIOD_LEN=30000",
+                     "SWIM_SUSPECT_TIMEOUT=90000",
+                     "DD_MASK=mgmt,io,md,epc,rebuild"]
+                )
+            else:
+                self.env_vars = BasicParameter(
+                    None,
+                    ["ABT_ENV_MAX_NUM_XSTREAMS=100",
+                     "ABT_MAX_NUM_XSTREAMS=100",
+                     "DAOS_MD_CAP=1024",
+                     "DD_MASK=mgmt,io,md,epc,rebuild"]
+                )
+
```

Comment:
I'd suggest the following changes to enhance maintenance and the updated python string compare:
```suggestion
            # Set extra environment variables for sockets provider
            default_env_vars = [
                "ABT_ENV_MAX_NUM_XSTREAMS=100",
                "ABT_MAX_NUM_XSTREAMS=100",
                "DAOS_MD_CAP=1024",
                "DD_MASK=mgmt,io,md,epc,rebuild"
            ]
            if default_provider == "ofi+sockets":
                default_env_vars.extend([
                    "FI_SOCKETS_MAX_CONN_RETRY=5",
                    "FI_SOCKETS_CONN_TIMEOUT=2000",
                    "CRT_SWIM_RPC_TIMEOUT=10",
                    "SWIM_PING_TIMEOUT=10000",
                    "SWIM_PROTOCOL_PERIOD_LEN=30000",
                    "SWIM_SUSPECT_TIMEOUT=90000",
                ])
            self.env_vars = BasicParameter(None, default_env_vars)

```